docs/tutorials/working-with-containers.md

```diff
@@ -108,24 +108,14 @@ docker run -d -p 8000:80 nginx
   </TabItem>
 </Tabs>
 
-You can then access the container via the browser here: [http://localhost:8080/](http://localhost:8080/).
+You can then access the container via the browser here: [http://localhost:8080/](http://localhost:8080/).  The port will be accessible on all network interfaces of the host machine.
```

Comment:
I added this comment about all network interfaces to help clarify the note below about how on Windows it's NOT accessible on all interfaces, but I don't know if it's actually true.  Can someone say for sure whether this is true or if it should be scoped down a bit.

Reply:
Hi @veleek, I tried a few things related to exposing the port on multiple network interfaces on macOS, Windows, and Linux machines. It looks like it's a windows specific behavior with Rancher Desktop. On macOS and Linux, I could access the ports on non localhost interfaces by default.

So, I suggest below changes to your proposed change.

- Remove this statement `The port will be accessible on all network interfaces of the host machine.` and clarify in the `Note`

- I suggest a rephrase of the `Note` as below..

> Note: By default the exposed ports are accessible on all network interfaces on macOS and Linux. However, on Windows, the exposed ports are currently only accessible through the `localhost` network interface (see issue [#1180](https://github.com/rancher-sandbox/rancher-desktop/issues/1180)). As a workaround, you can [configure a `portproxy` on the windows host](https://github.com/rancher-sandbox/rancher-desktop/issues/1180#issuecomment-1005514200) to expose the port to additional network interfaces.

Can you please make these changes?

Reply:
@gunamata - yup, I can make those changes when I'm back on a computer.  

Regarding removing the added sentence: in "the future" when the Windows code is fixed and behaves the same as other platforms we'll remove the note. Is it worth having something so that people know ports are exposed on all interfaces, or is that understood enough as is.

Reply:
thank you @veleek . 

In future, when we address the issue on windows then we may add that statement to make users aware that the ports are exposed on all interfaces.

Reply:
@gunamata - Changes complete!